W/Wayland/build_tarballs.jl

```diff
@@ -40,10 +38,10 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
```

Comment:
Please never remove compats like these, they're there for a reason: #3024

Reply:
Oops, sorry! I just thought those were only required to build the previous wayland version.